classes/book.rb

```diff
@@ -0,0 +1,17 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  # attr_reader :rentals
```

Comment:
- [OPTIONAL] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻 Since it's the only one you have I'm gonna approve this PR.